autonomous/autonomous.py

```diff
@@ -142,11 +142,15 @@ def turn_turret(self):
         self.turretTurn.setEncoderControl()
         self.turretTurn.setAngle(self.turretThreshold.leftLim + 97)
```

Comment:
An easy fix for the overshoot issue is that the 97 value should be reduced and we can let speed carry us to the position we want, if it overshoots consistently (in terms of angle, time, etc.)